core/src/main/java/io/questdb/cutlass/text/ParallelCsvFileImporter.java

```diff
@@ -1057,7 +1057,11 @@ private void phasePartitionImport() throws TextImportException {
 
     private void phasePrologue(byte phase) {
         phaseErrors = 0;
-        LOG.info().$("started  [phase=").$(phase).$(", file=`").$(inputFilePath).$('`').$(", workerCount=").$(workerCount).I$();
+        LOG.info()
```

Comment:
nit: would be nice to include import id into the logs here and in serial importer.